### ISSUE TITLE
Bakery has trouble running `npm install` from the parent directory, in Windows

### ISSUE THREAD

**Issue by alexweissman** [userfrosting/UserFrosting#742]
Bakery uses the [`--prefix` flag](https://docs.npmjs.com/misc/config#prefix) to tell `npm` which directory to target for installing node dependencies.

For whatever reason, this is working in *nix but not Windows:

```
> npm install
npm ERR! addLocal Could not install D:\dev\userfrosting
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\
node_modules\\npm\\bin\\npm-cli.js" "install" "--prefix" "D:\\dev\\userfrosting/
build"
npm ERR! node v6.11.0
npm ERR! npm  v3.10.10
npm ERR! code EISDIR
npm ERR! errno -4068
npm ERR! syscall read

npm ERR! eisdir EISDIR: illegal operation on a directory, read
npm ERR! eisdir This is most likely not a problem with npm itself
npm ERR! eisdir and is related to npm not being able to find a package.json in
npm ERR! eisdir a package you are trying to install.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\dev\userfrosting\npm-debug.log
```

`npm` docs don't seem to have any useful information.

For now, the workaround is to just run `npm install` directly in the `build/` subdirectory.

**Comment by alexweissman** [comment]
Possibly relevant: https://github.com/npm/npm/issues/17100

**Comment by passta** [comment]
When i use the workaround everything installs fine but after that i get the same error when i run "php bakery bake" 
Does the workaround fix this issue for you?  Because i am not able to open the testpage after the complete installation and setup of userfrosting

**Comment by lcharette** [comment]
Run bake, then from the `build/` directory, try `npm install` then `npm run uf-assets-install`. After that, you don't need to run bake again.

**Comment by alexweissman** [comment]
Ok, this should be fixed with the 4.1.1-alpha release.
